sdk/core/azure_core/src/auth.rs

```diff
@@ -78,6 +78,14 @@ impl AccessToken {
             expires_on,
         }
     }
+
+    /// Check if the token is expired
+    ///
+    /// This checks if the token is expiring within the next 30 seconds to
+    /// address clock skew
```

Comment:
Nit: punctuation looks more professional:

```suggestion
    /// Check if the token is expired.
    ///
    /// This checks if the token is expiring within the next 30 seconds to
    /// address clock skew.
```

Also, should we make the time configurable? Or not worth it until someone asks for it? Without overloads, we'd need a new method name, though, when that happens. Don't care too much, though: just putting out the question to see what you think. Could take an `Option<Duration>` and `unwrap_or()` to 30s.

Reply:
addressed

Reply:
If you're fine with that. It wasn't a "hard suggestion". Really just wondering out loud. :)